docs/docker_compose.md

```diff
@@ -36,7 +36,14 @@ That's it! Then run:
 $ tilt up
 ```
 
-Tilt will pick up your Docker Compose file and start running your srvices.
+Tilt will pick up your Docker Compose file and start running your services.
+
+Be aware of one important difference between `tilt up` and `docker-compose up`: Tilt
+will leave your services up when it exists. To turn the services down, run:
+
+```
+$ tilt down
+```
 
 ## Making Changes
 
```

Comment:
a little preamble here to make it clear that this is optional/an optimization? something to the effect of "you CAN just let `docker compose up` take care of building the images as well as running them. However, if you want to leverage Tilt's image-building expertise, you can let us build your images for you..."

Reply:
i like that! tweaked a bit and added